MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/SANSRunWindow.ui

```diff
@@ -93,7 +93,7 @@
        <string notr="true"/>
       </property>
       <property name="currentIndex">
-       <number>0</number>
+       <number>5</number>
```

Comment:
Just wanted to check this. I think this change sets the default tab index to `5`, doesn't it? Is that intended?